dspace-server-webapp/src/test/java/org/dspace/app/rest/RequestItemRepositoryIT.java

```diff
@@ -136,6 +144,8 @@ public void init()
     @Test
     public void testFindAll()
             throws Exception {
+        System.out.println("findAll");
```

Comment:
I believe our ITs will already print out the method name. I don't think these `System.out.println()` are necessary, as you should already see logs that say something like  "Running testFindAll()"  (I'm forgetting the exact log message though)